draft-modern-exchange-admin-center (#2213)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/exchange-admin-center.md

You use the Exchange admin center to manage email settings for your organization.

> ## Get to the New Exchange admin center 

Reviewer: ```suggestion
## Get to the new Exchange admin center 
```

> You must have [Microsoft 365 admin permissions](https://docs.microsoft.com/microsoft-365/admin/add-users/assign-admin-roles) to access the New Exchange admin center. For more information, see [Permissions in Exchange Online](permissions-exo/permissions-exo.md).

Reviewer: ```suggestion
You must have [Microsoft 365 admin permissions](https://docs.microsoft.com/microsoft-365/admin/add-users/assign-admin-roles) to access the new Exchange admin center. For more information, see [Permissions in Exchange Online](permissions-exo/permissions-exo.md).
```

Author: Hi Johan, the new version will be called "New Exchange Control Center". So I can't change it to "new". Thank you for the review. Currently its in review with the internal team too. So please don't merge it yet.

> 1. [Sign in](https://support.microsoft.com/office/e9eb7d51-5430-4929-91ab-6157c5a050b4) to Microsoft 365 or Office 365 using your work or school account (AI - Further down for the URL.we are stating to use credentials to login. Is that different from the work or school account? If not, then we need to maintain consistency in usage of terminology across), and then choose the **Admin** tile.

Reviewer: ```suggestion
1. [Sign in to Microsoft 365 or Office 365](https://support.microsoft.com/office/e9eb7d51-5430-4929-91ab-6157c5a050b4) using your work or school account, and then choose the **Admin** tile.
```

2. In the Microsoft 365 admin center, choose **Admin centers** \> **Exchange**.

    ![Modern EAC](media/EAC-selection.PNG)

> You can also get to the New Exchange admin center directly by using the URL [https://admin.exchange.microsoft.com](https://admin.exchange.microsoft.com) and sign in using your  credentials.

Reviewer: ```suggestion
You can also get to the New Exchange admin center directly by using the URL [https://admin.exchange.microsoft.com](https://admin.exchange.microsoft.com) and signing in using your credentials.
```

> You can also get to the New Exchange admin center directly by using the URL [https://admin.exchange.microsoft.com](https://admin.exchange.microsoft.com) and signing in with your  credentials.

Reviewer: ```suggestion
You can also get to the new Exchange admin center directly and sign in with your credentials [here](https://admin.exchange.microsoft.com).
```

> [!NOTE]

> >Be sure to use a private browsing session (not a regular session) to access the Exchange admin center using the direct URL. This will prevent the credential that you are currently logged on with from being used. To open an InPrivate Browsing session in Microsoft Edge or an incognito window in Google Chrome, press CTRL+SHIFT+N. To open an InPrivate Browsing session in Microsoft Edge Legacy, Internet Explorer, or a Private Browsing session in Mozilla Firefox, press CTRL+SHIFT+P.

Reviewer: ```suggestion
> Be sure to use a private browsing session (not a regular session) to access the Exchange admin center using the direct URL. This will prevent the credential that you are currently logged on with from being used. To open an InPrivate Browsing session in Microsoft Edge or an incognito window in Google Chrome, press CTRL+SHIFT+N. To open an InPrivate Browsing session in Microsoft Edge Legacy, Internet Explorer, or a Private Browsing session in Mozilla Firefox, press CTRL+SHIFT+P.
```

## New Exchange admin center features

Here's what the New Exchange admin center looks like.

![change admin centre features](media/Modern-Exchange-Admin-Center1.PNG)

![change admin detail pane](media/Modern-Exchange-Admin-Center2.PNG)

### Feature pane

Here are the features you'll find in the left-hand navigation.

|**Area**|**What you do here**|
|:-----|:-----|
|**Recipients**|View and manage your mailboxes (both user and shared mailboxes), groups, resource mailboxes, and contacts.|
|**Mail flow**|Manage remote domains, accepted domains, connectors, message tracing, alerts, and alert policies.|
|**Migration**|Migrate mailboxes in batches.|
|**Reports**|View reports on mail flow and migrated batches.|

> |**Insights**|Use the recommendations to discover trends and/or insights,  and take actions to fix issues related to mailbox and mail flow.|

Reviewer: ```suggestion
|**Insights**|Use the recommendations to discover trends and/or insights, and take actions to fix issues related to mailbox and mail flow.|
```

|**More features**|Access features and services from the classic exchange admin center.|

### Tabs

The tabs are your second level of navigation. Each of the feature areas contains various tabs, each representing a complete feature.

### Toolbar

When you click most tabs, you'll see a toolbar. The toolbar has icons that perform a specific action.

### List view

> When you select a tab, in most cases you'll see a list view. The list view in New Exchange admin center is designed to remove limitations that existed in Exchange Control Panel in the Classic Exchange admin center.

Reviewer: ```suggestion
When you select a tab, in most cases you'll see a list view. The list view in the New Exchange admin center is designed to remove limitations that existed in Exchange Control Panel in the Classic Exchange admin center.
```

### Details pane

When you select an item from the list view, information about that object is displayed in the details pane.

### Centers, Me tile, and Help

> The Centers tile allows you to change from one admin center to another. The Me tile allows you to sign out of the EAC and sign in as a different user. The Help bubble displays contextual help for fields when you create or edit an object.

Reviewer: ```suggestion
The Centers tile allows you to change from one admin center to another. The Me tile allows you to sign out of the New Exchange admin center and sign in as a different user. The Help bubble displays contextual help for fields when you create or edit an object.
```

### Toolbar

When you click most tabs, you'll see a toolbar. The toolbar has icons that perform a specific action.

### List view

> When you select a tab, in most cases you'll see a list view. The list view in Classic Exchange admin center is designed to remove limitations that existed in Exchange Control Panel.

Reviewer: ```suggestion
When you select a tab, in most cases you'll see a list view. The list view in the Classic Exchange admin center is designed to remove limitations that existed in Exchange Control Panel.
```
